src/GHC/Util/Brackets.hs

```diff
@@ -22,6 +27,13 @@ isOpApp' (LL _ OpApp{}) = True; isOpApp' _ = False
 isAnyApp' x = isApp' x || isOpApp' x
 isSection' (LL _ SectionL{}) = True; isSection' (LL _ SectionR{}) = True; isSection' _ = False
 
+-- See note [DA 'HasField' expressions] below.
+mod_records = mkModuleName "DA.Internal.Record"
```

Comment:
I like type signatures for top-level definitions :)

Reply:
Yeah. I'll come back to that today.